frontend-zs/src/components/projects/Home/Main.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import Axios from 'axios';
+import { Container, Row } from 'react-bootstrap'
+import { ProjectCardsList } from './ProjectCard';
+import { SearchBar } from './SearchBar';
+
+export function ProjectsList() {
+    const [allProjects, setAllProjects] = useState([])
+    const [projects, setProjects] = useState(allProjects)
```

Comment:
```suggestion
    const [displayProjects, setDisplayProjects] = useState(allProjects)
```

displayProjects might be more descriptive for the state. What do you think?

Reply:
Yup, good point. I'll change it

Reply:
Good point, I'll change it